cmd/exporter/main.go

```diff
@@ -49,14 +49,23 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	go func() {
 		defer cancel()
-		scrape, err := gocdexporter.NewScraper(&gocdexporter.Config{
+		conf := &gocdexporter.Config{
 			*ns, prometheus.DefaultRegisterer, *url, *user, *passwd, *agentMaxPages,
-		})
+		}
+		scrape, err := gocdexporter.NewScraper(conf)
 		if err != nil {
 			log.Fatal(err)
 		}
 		ticker := time.NewTicker(dur)
 		defer ticker.Stop()
+		errorGauge := prometheus.NewGaugeVec(
```

Comment:
Maybe this should be a `NewCounterVec` so that the history between the scrapes can be preserved?